jetstream/probe_sets.py

```diff
@@ -46,9 +44,10 @@ def columns_for_scalar(self, scalar_name: str) -> List[str]:
 
 
 @attr.s(auto_attribs=True, kw_only=True, slots=True, frozen=True)
-class FeatureScalarTelemetry:
+class TelemetryScalarProbe:
     kind: ClassVar[str] = "scalar"
     name: str
+    event_category: str
```

Comment:
```suggestion
```

Reply:
It looks like this field is there for both probe types based on Experimenters schema definition: https://github.com/mozilla/experimenter/blob/7e0635a58f9cb81606985a05a6ec3a03d827f50b/app/experimenter/docs/openapi-schema.json#L3595

Reply:
It sounds like the Experimenter definition is wrong. Would you mind filing an issue?

Reply:
Will do